src/media/css/feed/feed.styl

```diff
@@ -182,6 +183,13 @@
             text-overflow: ellipsis;
             white-space: nowrap;
         }
+        .desc {
+            font-size: 16px;
```

Comment:
It'd be consistent with everything else. We can update the mixin later if we have to. The type in the PR doesn't match that spec atm anyways